packages/react-chat/src/components/Chat/index.tsx

```diff
@@ -104,10 +104,10 @@ const Chat: React.FC<ChatProps> = ({
           <Spacer />
           {!!timestamp && !!state.session.turns.length && <SessionTime>{timestamp}</SessionTime>}
           {children}
-          {hasEnded && !!state.session.turns.length && <Status>You have ended the chat</Status>}
+          {hasEnded && !!state.session.turns.length && <Status>The chat has ended</Status>}
```

Comment:
most of the time the chat has ended, it is not because the user ended the chat, it is because the bot is just done talking.